getting_started/setup/setup_ktest_environment.rst

```diff
@@ -59,15 +59,71 @@ working kernel if your changes fail to boot.
 
    # Use your text editor of choice.
    sudo emacs /etc/default/grub
+   sudo update-grub
```

Comment:
Should we put what we edit in /etc/default/grub in between?

Reply:
Logically yes but that would consume two  `code-block`s for two lines, a bit too verbose IMHO. I'm assuming the reader's look-ahead buffer is at least 2 lines long...